packages/frontpage/middleware.ts

```diff
@@ -92,8 +103,10 @@ export async function middleware(request: NextRequest) {
       // Logout and show error
       console.log("session corrupt, logging out", result);
       await signOut();
-      const response = NextResponse.redirect(new URL("/login", request.url));
-      deleteAuthCookie(response.cookies);
+      const response = NextResponse.redirect(
+        new URL("/login", request.url),
+        NextResponse.next(),
+      );
```

Comment:
Merging the redirect response like this allows us to remove the need for `deleteAuthCookie` as we can just rely on `cookies().delete()` to be called from inside signOut.